States/SfwEntryState.cs

```diff
@@ -34,21 +33,6 @@ public Task Enter(InteractionContext c)
                 .Build());
         }
 
-        public Task OnMessageReceived(MessageInteractionContext c)
-        {
-            var venue = c.Session.GetItem<Venue>("venue");
-
-            if (c.Prediction.TopIntent == IntentNames.Response.Yes)
-                venue.Sfw = true;
-            else if (c.Prediction.TopIntent == IntentNames.Response.No)
-                venue.Sfw = false;
-            else
-                return c.Interaction.Channel.SendMessageAsync(MessageRepository.DontUnderstandResponses.PickRandom());
-
-            if (c.Session.GetItem<bool>("modifying"))
-                return c.Session.MoveStateAsync<ConfirmVenueState>(c);
-            return c.Session.MoveStateAsync<CategoryEntryState>(c);
-        }
     }
 

```

Comment:
Left over from when you had to type "yes" or "no" to the N/SFW question. Removed it now so you can only answer via the button.